### DISCUSSION TITLE
Student program

### DISCUSSION THREAD

**Discussion by eliasto** [community/community#23003] [New to GitHub]
<p>Hi everyone! I apply for the student program and I was refused! Indeed, I was noticed that I need to use my school-issed email to apply, but I used it! So where is the error? I have waiting 10 days to got a refused…</p>
<p>Thanks for your help,</p>
<p>Elias.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/eliasto">@eliasto</a>, Thank you for being here! Please message us at <a href="mailto:education@github.com">education@github.com</a>, and we’ll see what we can do.</p>
